lib/membrane/rtp/tcp_depacketizer.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Membrane.RTP.TCP.Depacketizer do
```

Comment:
It was kind of counterintuitive to me that Depacketizer is reponsible for encapsulation, even though it technically is correct. What do you think about Encapsulator and Decapsulator?

Reply:
Personally, I would probably stay with Packetizer/Depacketizer or Payloader/Depayloader, but I don't mind changing that

Reply:
Hmm, actually is it correct? I think of depacketizing as of getting out of packets, which in this case would be removing the two-byte header with packet length…

Reply:
the RFC calls it framing: https://datatracker.ietf.org/doc/html/rfc4571 however as I read it, I understand that by framing they mean adding the 2 byte header, not the other way

Reply:
packetizing converting to packets from a continuous stream of data (TCP does not really have a concept of packets)